docs/connectors/rhoas-cli-getting-started-connectors/README.adoc

```diff
@@ -157,9 +161,11 @@ A {connectors} namespace hosts your {connectors} instances.
 
 The namespace that you use depends on your OpenShift Dedicated environment.
 
-If you're using a trial cluster in your own OpenShift Dedicated environment:: The namespace is created when you add the {product-connectors} service to your trial cluster, as described in https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/15a79de0-8827-4bf1-b445-8e3b3eef7b01[Adding the Red Hat OpenShift {connectors} add-on to your OpenShift Dedicated trial cluster^]. Your OpenShift Dedicated trial cluster namespace is active for 60 days.
+If you're using the hosted preview environment:: You must create a preview namespace before you can create {connectors} instances. A preview namespace is active for 48 hours.
+
+If you're using a trial cluster in your own OpenShift Dedicated environment:: The namespace is created when you add the {product-connectors} service to your trial cluster, as described in https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/15a79de0-8827-4bf1-b445-8e3b3eef7b01[Adding the Red Hat OpenShift {connectors} add-on to your OpenShift cluster^]. Your OpenShift Dedicated trial cluster namespace is active for 60 days.
```

Comment:
This link will not work upstream. Recommend using an attribute for the link instead. Looks like `{addon-url-connectors}` is the correct one for this guide, and has already been set downstream, but is missing upstream?